### PR TITLE
Add Carrierwave gem.Add an attachments column to posts table. 

### PR DESCRIPTION
**Pull request by vakyl9r**

### CODE REVIEW

**Comment by vakyl9r** [comment]
Added an ability to attach files to post.
**FOR NOW FILES DISPLAYED IN IMAGE TAGS** just for testing purposes.

File: Gemfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ gem 'sprockets-rails', :require => 'sprockets/railtie'
 gem 'bcrypt', '~> 3.1', '>= 3.1.11'
 gem 'bootstrap-sass'
 gem 'string_master'
+gem 'carrierwave', github: 'carrierwaveuploader/carrierwave'
```
Comment (Magons) [inline Gemfile]: remove `github: 'carrierwaveuploader/carrierwave'`

File: app/controllers/attachments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class AttachmentsController < ApplicationController
+  before_action :set_post
+  respond_to :json, :html
+
+
```
Comment (Magons) [inline app/controllers/attachments_controller.rb]: remove empty line

File: app/controllers/attachments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  private
+
+  def set_post
+    @post = Post.find(params[:post_id])
+  end
+
+  def remove_attachment_at_index(index)
+    remain_attachments = @post.attachments # copy the array
+    deleted_attachment = remain_attachments.delete_at(index) # delete the target attachment
+    deleted_attachment.try(:remove!) # delete attachment from S3
```
Comment (Magons) [inline app/controllers/attachments_controller.rb]: remove this. You don't use S3
instead try use `&` in future

File: app/views/posts/show.html.haml [hunk 4]
```diff
@@ -1,4 +1,28 @@
...
+    .form-group
+      = image_tag attachment.url, class: 'img-thumbnail'
+= link_to 'Редактировать', edit_post_path(@post)
+|
+= link_to 'Назад', root_path
+.list-group.comments
+  %a.list-group-item.active
+    = 'Комментарии:'
+  - @post.comments.each do |comment|
+    - if comment.body?
```
Comment (Magons) [inline app/views/posts/show.html.haml]: do rebase

File: Gemfile (hunk 1 above)
Comment (vakyl9r) [reply Gemfile]: @Magons it's necessary for multiple file upload 
https://github.com/carrierwaveuploader/carrierwave#multiple-file-uploads

**Comment by vakyl9r** [comment]
@Magons Fixed

**Comment by vakyl9r** [comment]
@Magons don't merge just yet. Still trying out different options.

**Comment by Magons** [comment]
@vakyl9r @AlexSukov create label 'to verify' and use it when PR ready to check

**Comment by Magons** [comment]
@vakyl9r can I verify this? Don't see label

**Pull request merged**